src/node_sqlite.cc

```diff
@@ -573,19 +642,70 @@ class BackupJob : public ThreadPoolWork {
     resolver->Reject(env()->context(), e).ToChecked();
   }
 
+  inline MaybeLocal<Object> CreateAbortError(
+      Isolate* isolate, const char* message = "The operation was aborted") {
+    Local<Value> cause;
+
+    // Extract the reason from the abort signal if available
+    if (!abort_signal_.IsEmpty()) {
+      HandleScope scope(isolate);
+      Local<Object> signal = abort_signal_.Get(isolate);
+      Local<String> reason_key =
+          String::NewFromUtf8(isolate, "reason").ToLocalChecked();
```

Comment:
Avoid using `ToLocalChecked`. If this returns an empty maybe, then return an empty `MaybeLocal<Object>`

Reply:
Also, we have `env->reason_string()` already.